elastic/integrations
# Removing detection rules from the `ded` package

## What does this PR do?
This PR moves detection rules associated with Data Exfiltration Detection (ded) out of the integration package. 

## Checklist

## How to test this PR locally
- Spin up a local Elasticsearch cluster
- Verify that the package version shown is 2.0.0, Subscription is `platinum` and License is `Elastic-2.0`
- Install the Data Exfiltration Detection assets
- Verify that no detection rules with the `Data Exfiltration` tag show up on the Rules Management page under Security
- Verify that the job IDs have been updated when you click on the `Data Exfiltration` card under Anomaly Explorer

## Related Issues
Not linking since these are internal issues

## Screenshots
- Version shows 2.0 while installing assets
<img width="1267" alt="Screen Shot 2023-09-22 at 10 39 39 AM" src="https://github.com/elastic/integrations/assets/30438249/920efd63-b11b-4d07-929e-4e6849c77e24">

- No rules appear upon installing assets
<img width="1507" alt="Screen Shot 2023-09-22 at 10 47 16 AM" src="https://github.com/elastic/integrations/assets/30438249/cac9cd07-226f-4caf-a969-1c6fc8452cd3">

- Job IDs have been updated
<img width="730" alt="Screen Shot 2023-09-22 at 10 43 21 AM" src="https://github.com/elastic/integrations/assets/30438249/b8c9bbc9-0f85-4cd5-85d8-86067b3313db">
